index.js

```diff
@@ -13,5 +24,23 @@ module.exports = (api, options) => {
     webpackConfig.output.libraryTarget("umd");
 
     webpackConfig.set("devtool", "sourcemap");
+
+    webpackConfig
+      .plugin("SystemJSPublicPathWebpackPlugin")
+      .use(SystemJSPublicPathWebpackPlugin, [
+        {
+          rootDirectoryLevel: 2,
```

Comment:
vue-cli projects have a `/js` directory that needs to be removed for correct public path